smart_proxy_monitoring.gemspec

```diff
@@ -15,5 +15,5 @@ Gem::Specification.new do |s|
   s.add_dependency 'json'
   s.add_dependency 'rest-client'
 
-  s.required_ruby_version = '>= 2.5', '< 3'
+  s.required_ruby_version = '>= 3.0'
```

Comment:
We don't drop Ruby 2.7 support, but we can drop Ruby 2.5. I'd also prefer an upper bound of `< 4` just to be sure. Even though Ruby really doesn't stick to semver and just breaks in minor versions all the time, it's still better to at least have some upper bound.